### DISCUSSION TITLE
Attach clients into the stack area when you toggle a new tag into view

### DISCUSSION THREAD

**Discussion by 3333juul** [awesomeWM/awesome#3897] [Q&A]
Hi, I'm trying to replicate this behaviour in awesome:
https://dwm.suckless.org/patches/tagintostack/

,,`tagintostack` new clients attach into the stack area when you toggle a new tag into view. This means your master area will remain unchanged when toggling views."

Any advice how could I do this?

**Comment by Aire-One** [comment]
Hello,

Some times ago, the default `rc.lua` had this section in:

```lua
client.connect_signal("manage", function (c)
    -- Set the windows at the slave,
    -- i.e. put it at the end of others instead of setting it master.
    -- if not awesome.startup then awful.client.setslave(c) end

    if awesome.startup
      and not c.size_hints.user_position
      and not c.size_hints.program_position then
        -- Prevent clients from being unreachable after screen count changes.
        awful.placement.no_offscreen(c)
    end
end)
```

It was removed by https://github.com/awesomeWM/awesome/commit/067bcaca60ed138f016388be34675e22d0421bc2.

The interesting part for your use case is the `awful.client.setslave(c)`. Nowadays, there are multiple places where you can place it. `Ruled.client` are definitely a good candidate. The client `"request::manage"` signal that replace the old `"manage"` signal also works.
